src/tyr/isochrone_serializer.cc

```diff
@@ -169,7 +186,7 @@ std::string serializeGeoTIFF(Api& request, const std::shared_ptr<const GriddedDa
   char** geotiff_options = NULL;
   geotiff_options = CSLSetNameValue(geotiff_options, "COMPRESS", "PACKBITS");
 
-  // TODO: check if there's an easy way to only instantiate the driver once
+  gdal_singleton_t::get();
```

Comment:
this first time this is called (per process) the constructor above is called and the driver registration happens. subsequent calls are no-ops